equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Customer organization identifier",
+		},
+		"organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Customer organization name",
+		},
+		"global_org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization identifier",
+		},
+		"global_organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization name",
+		},
+		"global_cust_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global Customer organization identifier",
+		},
+	}
+}
+
+func readNotificationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Notification Type",
+		},
+		"send_interval": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Send interval",
+		},
+		"emails": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Array of contact emails",
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
+	}
+}
+
+func readErrorAdditionalInfoSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"property": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Property at which the error potentially occurred",
+		},
+		"reason": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Reason for the error",
+		},
+	}
+}
+
+func readOperationalErrorSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"error_code": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Error  code",
+		},
+		"error_message": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Error Message",
+		},
+		"correlation_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "CorrelationId",
+		},
+		"details": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Details",
+		},
+		"help": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Help",
+		},
+		"additional_info": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Pricing error additional Info",
+			Elem: &schema.Resource{
+				Schema: readErrorAdditionalInfoSch(),
+			},
+		},
+	}
+}
+
+func readOperationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"provider_status": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Connection provider readiness status",
```

Comment:
Some potential status values would be helpful here and on `equinix_status` if they are enumerable.

Reply:
taken care